packages/dashboards/addon/templates/dashboards/dashboard/widgets/widget.hbs

```diff
@@ -1,114 +1,120 @@
-            >
-              <NaviIcon @icon="download" class="navi-report-widget__icon" />
-              Export
-              <EmberTooltip>
-                {{if @model.validations.isTruelyValid "Export the report" "Run a valid report to enable export"}}
-              </EmberTooltip>
-            </ExportAction>
-          {{/let}}
-        {{/let}}
-      </li>
-
-      <li class="navi-report-widget__action">
-        <LinkTo 
-          @route="dashboards.dashboard.widgets.widget.clone-to-report" 
-          class="navi-report-widget__action-link button is-text is-medium"
-        >
-          <NaviIcon @icon="copy" class="navi-report-widget__icon" />
-          Clone As Report
-        </LinkTo>
-      </li>
+  <div class="container-full p-y-10">
+    {{!-- Breadcrumb --}}
+    <div class="row">
+      <div class="col-1-1">
+        <div class="navi-report-widget__breadcrumb">
+          {{#if (feature-flag "enableDirectory")}}
+            <LinkTo @route="directory.my-data" class="navi-report-widget__breadcrumb-link">Directory</LinkTo>
+          {{else}}
+            <LinkTo @route="dashboards" class="navi-report-widget__breadcrumb-link">Dashboards</LinkTo>
+          {{/if}}
+          <NaviIcon @icon="angle-right" />
+          {{#with (model-for "dashboards.dashboard") as |parentDashboard|}}
+            {{#if this.parentQueryParams}}
+              <LinkTo @route="dashboards.dashboard" @model={{parentDashboard.id}} @query={{hash filters=this.parentQueryParams.filters}} class="navi-report-widget__breadcrumb-link">
+                {{parentDashboard.title}}
+              </LinkTo>
+            {{else}}
+              <LinkTo @route="dashboards.dashboard" @model={{parentDashboard.id}} class="navi-report-widget__breadcrumb-link">
+                {{parentDashboard.title}}
+              </LinkTo>
+            {{/if}}
+          {{/with}}
+          <NaviIcon @icon="angle-right" />
+        </div>
+      </div>
+    </div>
 
-      {{#unless @model.isNew}}
-        {{!-- Share only enabled on a Valid widget --}}
-        <li class="navi-report-widget__action">
-          <CommonActions::Share
-            @disabled={{not @model.request.validations.isTruelyValid}}
-            class="button is-text is-medium"
+    {{!-- Header --}}
+    <div class="navi-report-widget__header">
+      <div class="row align-items-center">
+        <div class="xs-col-10-10 sm-col-10-10 col-4-10 p-x-0">
+          <span class="navi-report-widget__title">
+            {{#if @model.dashboard.canUserEdit}}
+              <EditableLabel
+                class="navi-report-widget__edit-title"
+                @value={{@model.title}}
+                @onChange={{route-action "updateTitle"}}
+              />
+            {{else}}
+              <span>{{@model.title}}</span>
+            {{/if}}
+          </span>
+        </div>
+        <div class="xs-col-10-10 sm-col-10-10 col-6-10 p-x-0">
```

Comment:
Looks like this is missing the equivalent [wrapper](https://github.com/yavin-dev/framework/blob/22838a752678a3dc92fc687de2d858d4aa4c0add/packages/reports/addon/components/report-header.hbs#L39) which [aligns them to the end](https://github.com/yavin-dev/framework/blob/2a3c6cf64e28a8e21fb8c4b9e281d5e76d28a20e/packages/reports/app/styles/navi-reports/common/report.scss#L30)